package.json

```diff
@@ -60,6 +61,11 @@
     "sanitize-html"
   ],
   "pre-commit": {
-    "run": ["lint"]
+    "run": [
+      "lint"
+    ]
+  },
+  "dependencies": {
+    "package.json": "^2.0.1"
```

Comment:
Also what's going on with this? It looks like it has security vulnerabilities

Reply:
I could've been when I did npm i package.json. I think we can leave it out of the merge